reference/functions/content.md

```diff
@@ -1,41 +1,34 @@
 # `content()`
 
-The `content` XPath function is a convenient function to access
-contents from inside the [flow](../flow/Flow.html). However, as an
-XPath function you may use it in expressions outside the flow, too.
+The `content` function is a convenient function to access
+contents from inside the [flow](../flow.md).
 
-```markup
+```xml
 <flow>
-  <default-request />
   <if test="content()/html/head/title = 'Thank you'">
-    ...
+    …
   </if>
-  ...
 </flow>
 ```
 
 When called without parameter, the function returns the main content DOM from
 `fit://request/content`.
 
-The following XPath expressions are equivalent:
+The following `test` expressions are equivalent:
 
 * `content()/html/head/title`
 * `content('')/html/head/title`
 * `fit-document('fit://request/content')/html/head/title`
 * in XSLT you may use the standard function `document('fit://request/content')/html/head/title`
 
-;;block-note
-
 Be aware that `fit://request/content` and `fit://request/content/main` are not
 the same thing! While the first starts as a copy of the latter, flow actions may
 alter `fit://request/content` throughout the request.
 `fit://request/content/main` holds the plain text response body and usually
 remains unchanged. Thus, `content()` and `content('main')` are not necessarily
 equivalent.
 
-block;;
-
-Optionally, a [content ID](../flow/FlowAction_Requests.html) can
+Optionally, a [content ID](../flow.md) can
```

Comment:
Wo in flow.md steht etwas zu content ID?

Reply:
Hm, das sollte wohl eher `actions/requests.md` heißen…